.github/std-unexploded.yml

```diff
@@ -51,6 +51,12 @@ jobs:
       #   with:
       #     nixbuild_ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
       #     generate_summary_for: job
+
+      - name: Show commit
```

Comment:
Could make this an `Action` to DRY

Reply:
Good idea, addressed in [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/956e95b3e27cf401bb8530f6cd5d058b6824ff00..2f6a6a91806296061f82aa0295561b2f492981f5). Let's see if it works as expected :hourglass:

Reply:
Didn't work, maybe [this](https://github.com/input-output-hk/cardano-js-sdk/compare/2f6a6a91806296061f82aa0295561b2f492981f5..a8fbae2e719dbd7d48171892aa770a2ba28dbc4c) does.

Reply:
Turn's out this is not going to work:
> Did you forget to run actions/checkout before running your local action?

I need to revert to the original version.

Reply:
Well, in the original form, it at least works:
![image](https://github.com/input-output-hk/cardano-js-sdk/assets/7548295/48b2b984-6d79-4401-8fe8-9f9e27378f86)
:smile: